metrics/stats.go

```diff
@@ -7,7 +7,7 @@ import (
 	"time"
 )
 
-type Stats struct {
+type GoStats struct {
```

Comment:
__[golint]__
* [golint] exported type GoStats should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->